packages/wallets/webpack.config.js

```diff
@@ -188,6 +196,7 @@ module.exports = function (env) {
                 'react-dom': true,
                 classnames: true,
                 'react-router-dom': true,
+                moment: true,
```

Comment:
core package already has moment included, so we exclude it in wallets vendors chunk